couchdb_cleaner.py

```diff
@@ -0,0 +1,185 @@
+import ast
+import sys
+from openprocurement.api.utils import read_yaml
+from couchdb.client import Server
+from couchdb.http import Unauthorized
+
+
+APP_META = './etc/app_meta.yaml'
+config = read_yaml(APP_META)['config']
+
+database_server = config['database']['couchdb.url']
+database_name = config['database']['couchdb.db_name']
+
+
+def base_view(items):
+    """Returns a javascript function for filtering records in database
+    :param items: An string representation of a logic statement in javascript
+    :return: An string representation of javascript function
+    """
+    return '''function(doc) {
+         if(%s) {
+            emit(doc);
+         }
+     }
+''' % items
+
+
+def make_statement(field, value, sub_key=None, sub_query=False):
+    """Returns string representation of a javascript object
+       and his property, and checks if a property is equal to a value
+    :param field: a property of a object
+    :param value: value which will be checked
+    :param sub_key: a sub property of a property
+    :param sub_query: if True, return key, and a sub key of object
+    :return: string representation of a javascript object and his keys
+
+    Example without subquery
+    >>> make_statement('_id', '12345')
+    doc._id == '12345'
+
+    Example with subquery
+    >>> make_statement('dog', 'Marti', 'pets', sub_query=True)
+    doc.pets.dog == 'Marti'
+    """
+    if sub_query:
+        return "doc.{}.{} == '{}'".format(sub_key, field, value)
+    return "doc.{} == '{}'".format(field, value)
+
+
+def convert_to_dict(data):
+    """Convert a string representation of dictionary
+       into python dictionary type
+    :param data: An string representation of a dictionary
+    :return: dictionary object
+
+    >>> convert_to_dict("{'_id': '1234'}")
+    {'_id': '1234'}
+    """
+    return ast.literal_eval(data)
+
+
+def has_sub_query(query):
```

Comment:
No, I think this function make sense, because we divide our code into logical parts, and it seems more intuitive. Of course , we able to do all this flow in `make_query` function, but it isn't а good way in this case. IMHO